x-pack/metricbeat/module/appsearch/docker-compose.yml

```diff
@@ -2,8 +2,11 @@ version: '2.3'
 
 services:
   appsearch:
+    image: docker.elastic.co/integrations-ci/beats-appsearch:${APPSEARCH_VERSION:-7.5.0}-1
     build:
       context: ./_meta
+      args:
+        APPSEARCH_VERSION: ${APPSEARCH_VERSION:-7.5.0}
```

Comment:
The default version should be 8.0.0-SNAPSHOT, so to test against the app-search master branch.

Reply:
Appsearch version is set to 7.5.0 also in master, we will review it as a follow up.